.github/workflows/build.yaml

```diff
@@ -0,0 +1,149 @@
+          echo "::set-output name=CACHE_KEY_BUILD::${{ hashFiles('**/Dockerfile', '**/*.py', '**/*.sh', '**/*.yaml') }}"
+          echo "::set-output name=CACHE_KEY_CLIENT_BUILD::${{ hashFiles('**/client/Dockerfile', '**/prepare_client.sh', '**/*.yaml') }}"
+          echo "::set-output name=CACHE_KEY_NODE_BUILD::${{ hashFiles('**/node/Dockerfile', '**/prepare_node.sh', '**/*.yaml') }}"
+          echo "::set-output name=PACKAGE_VERSION::1.0.0"
+          echo "::set-output name=PACKAGE_NAME::sovrin"
+
+  build-package:
+    needs: setup-workflow
+    runs-on: ubuntu-latest
+    env:
+      PACKAGE_NAME: ${{ needs.setup-workflow.outputs.PACKAGE_NAME }}
+      PACKAGE_VERSION: ${{ needs.setup-workflow.outputs.PACKAGE_VERSION }}
+      CACHE_KEY_BUILD: ${{ needs.setup-workflow.outputs.CACHE_KEY_BUILD }}
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Try load from cache.
+        id: cache-artifact
+        uses: actions/cache@v2
+        with:
+          path: build-scripts/ubuntu-2004/sovrin_*.deb
+          key: ${{ env.CACHE_KEY_BUILD}}
+      - name: Build package
+        if: steps.cache-artifact.outputs.cache-hit != 'true'
+        run: |
+          cd build-scripts/ubuntu-2004
+          # TODO: Cache the image, optimize scripts, remove default versions
+          ./build-sovrin-docker.sh "${PWD}/../.." "${PACKAGE_VERSION}" "${PWD}"
+      - name: Archive package
+        uses: actions/upload-artifact@v2
+        with:
+          name: sovrin-build-cd
+          path: build-scripts/ubuntu-2004/sovrin_*.deb
+
+  run-system-tests:
+    needs: [setup-workflow, build-package]
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        test: ["test_ledger.py"]
+      #         test: ['test_ledger.py', 'TestViewChangeSuite.py', 'TestConsensusSuite.py', 'test_state_proof.py']
+      fail-fast: false
+    env:
+      PACKAGE_NAME: ${{ needs.setup-workflow.outputs.PACKAGE_NAME }}
+      PACKAGE_VERSION: ${{ needs.setup-workflow.outputs.PACKAGE_VERSION }}
+      CACHE_KEY_CLIENT_BUILD: ${{ needs.setup-workflow.outputs.CACHE_KEY_CLIENT_BUILD }}
+      CACHE_KEY_NODE_BUILD: ${{ needs.setup-workflow.outputs.CACHE_KEY_NODE_BUILD }}
+      GITHUB_REPOSITORY_NAME: ${{ needs.setup-workflow.outputs.GITHUB_REPOSITORY_NAME }}
+      #       PACKAGE_URL: ${{ needs.publish-to-github.outputs.PACKAGE_URL }}
+
+      # Client image
+      CLIENT_REPO_COMPONENT: master
+      LIBINDY_VERSION: 1.10.0~1171
+
+      # Node image
+      NODE_REPO_COMPONENT: rc
+      LIBINDY_CRYPTO_VERSION: 0.4.5
+      PYTHON3_LIBINDY_CRYPTO_VERSION: 0.4.5
+      PYTHON3_PYZMQ_VERSION: 18.1.0
+      INDY_PLENUM_VERSION: 1.12.2
+      INDY_NODE_VERSION: 1.12.2~rc1
+      SOVRIN_VERSION: ${{ needs.setup-workflow.outputs.PACKAGE_VERSION }}
+      SOVRIN_INSTALL: true
+
+    steps:
+      - name: Check out test automation
+        uses: actions/checkout@v2
+        with:
+          repository: devinleighsmith/indy-test-automation
```

Comment:
this will need to be fixed when the corresponding PR for this is merged.

Reply:
Looks like the PR was merged.